packages/mdc-textfield/icon/_mixins.scss

```diff
@@ -59,25 +59,15 @@
 
 // Private mixins
 
-@mixin mdc-text-field-icon-horizontal-position_($position-property, $position) {
-  @if ($position-property == left) {
-    .mdc-text-field__icon--leading {
-      @include mdc-rtl-reflexive-position($position-property, $position);
-    }
-  } @else {
-    .mdc-text-field__icon--trailing {
-      @include mdc-rtl-reflexive-position($position-property, $position);
-    }
-  }
-}
-
-@mixin mdc-text-field-icon-horizontal-position-two-icons_($position-left, $position-right) {
```

Comment:
Why was this removed?

Reply:
It's doing the exact same thing as the other two mixins that set the leading/trailing positions.

It was necessary before because we didn't have separate classes for each one. It was juggling setting left vs right on `.mdc-text-field__icon` depending on if there was `with-leading`, `with-trailing`, or both on the host element.

Reply:
Looks like I might need to add some logic back in to fix the screenshot tests to the correct state. It's using the wrong position for the trailing icon that we have b/146057700 for